index.html

```diff
@@ -423,6 +423,29 @@ <h2>
                       abort these steps. 
                     </p>
                   </li>
+                  <li>
+                    <p>Let <var>topContext</var> be the <a>current settings object</a>'s's [=relevant global object=]'s
+                      [=associated Document=]'s [=top-level browsing context=].</p>
```

Comment:
HTML demoted browsing context (still there but meant for highly specifc behavior), in favor of new language. E,g, see mediacapture-main:

If mediaDevices's [relevant global object](https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global)'s [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigable)'s [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversable) has [system focus](https://html.spec.whatwg.org/multipage/#tlbc-system-focus), [do X]

Reply:
Switched to traversable and system focus.